Minifigures.UI/src/app/components/edit-boardgame/edit-boardgame.component.html

```diff
@@ -0,0 +1,26 @@
+<div *ngIf="boardgame">
+    <h1>{{ boardgame.title }}</h1>
+    <div>
```

Comment:
🌱 For the future we can create multiple inputs as FormControlls on a FormGroup - that will allow us to add some nice vallidation